{{cookiecutter.project_slug}}/config/settings/local.py

```diff
@@ -50,7 +50,7 @@
 
 # django-debug-toolbar
 # ------------------------------------------------------------------------------
-MIDDLEWARE_CLASSES += ('debug_toolbar.middleware.DebugToolbarMiddleware',)
+MIDDLEWARE += ('debug_toolbar.middleware.DebugToolbarMiddleware',)
```

Comment:
Sigh. I know this is necessary, but I'm not the biggest fan of the middleware changes. 😩